lambdas/s3hash/Dockerfile

```diff
@@ -0,0 +1,43 @@
+ARG FUNCTION_DIR="/function"
+
+FROM python:3.8-slim-bullseye@sha256:5f399249dc208063801e3316bcf98b927be8702ca29023317b792b2bc7c52884 AS base-image
```

Comment:
I think we prefer AWS based images now, so probably should use https://hub.docker.com/r/amazon/aws-lambda-python/tags